src/help/zaphelp/contents/start/concepts/alerts.html

```diff
@@ -23,7 +23,7 @@
 </table>
 
 <p>
-Alerts can be raised either via <a href="ascan.html">active scanning</a> or manually using the
+Alerts can be raised via <a href="ascan.html">active scanning</a>, <a href="pscan.html">passive scanning</a>, scripts, or manually using the
```

Comment:
Any extension can raise alerts, maybe worth saying that other "components" (for example, HttpsInfo add-on ;) might raise alerts as well?